tests/unit/utils/test_utils_url.py

```diff
@@ -45,7 +45,6 @@ def test_is_pdf_link_handles_empty_requests():
         assert not is_pdf_link('http://example.org/empty-pdf')
 
 
-@pytest.mark.vcr()
```

Comment:
this might fail, since it will try to reach arxiv server https://github.com/inspirehep/inspire-next/blob/a1e6356e2ed30a764825d6e5fa875a16efa2d3e0/inspirehep/utils/url.py#L64